lib/nylas/api.rb

```diff
@@ -40,6 +40,16 @@ def accounts
       @accounts ||= Collection.new(model: Account, api: as(client.app_secret))
     end
 
+    # @return [Collection<Folder>] A queryable collection of Accounts
+    def folders
+      @accounts ||= Collection.new(model: Folder, api: self)
```

Comment:
Hmm, not sure I understand the `@accounts ||=` part.

Reply:
Copy-paste error. Good catch!